lr-cli/cortx_setup/network.py

```diff
@@ -0,0 +1,78 @@
+import argparse
+import logging
+
+from provisioner.commands import PillarSet
+from provisioner.salt import local_minion_id
+
+logger = logging.getLogger(__name__)
+
+
+class Network():
+
+    """Configure network"""
+
+    def config(self, transport_type=None, interface_type=None,
+            network_type=None, interfaces=None
+    ):
+        """Update provided network details in pillar data"""
+
+        node_id = local_minion_id()
+        if transport_type is not None:
+            logger.info("Updating transport type in pillar data")
+            PillarSet().run(
+                f'node_info/{node_id}/network/data/transport_type',
+                f'{transport_type}',
+                targets=node_id,
+                local=True
+            )
+
+        if interface_type is not None:
+            logger.info("Updating interface type in pillar data")
+            PillarSet().run(
+                f'node_info/{node_id}/network/data/interface_type',
+                f'{interface_type}',
+                targets=node_id,
+                local=True
+            )
+
+        if network_type is not None:
+            if interfaces is not None:
+                logger.info(
+                    f"Updating interfaces for {network_type} network "
+                    "in pillar data"
+                )
+                PillarSet().run(
+                    f'node_info/{node_id}/network/{network_type}/intefaces',
```

Comment:
Do we need interface type here? (interface_type: public/private)
Network type is Mgmt/Data, so we need  a parameter to capture the interface type.
In this context, interface type has to be implicit parameter inside network.

Reply:
Addressed